### DISCUSSION TITLE
Saving session baseline entities

### DISCUSSION THREAD

**Discussion by Euan-p** [BottlecapDave/HomeAssistant-OctopusEnergy#1037] [Q&A]
Since updating to v13, the new saving session baseline entities (both export and import) have been showing 'unknown'. I tried looking on the website but couldn't find it on there. I've enabled the data last retrieved entity and it is updating, despite the actual data not being retrieved. I haven't made a bug report because I wasn't sure if this was normal or not, is it something like it only populates the entity when there's a session planned? Anyone know if I should make a bug report or if I can fix it? Thanks.

**Comment by BottlecapDave** [comment]
This is entirely normal because there are no active saving sessions. Once a joined saving session becomes available, it will be populated with the target baseline to the best of the available data.

**Comment by Euan-p** [comment]
Ok, thanks I wasn't sure if it was something like that or not. Thanks for the great integration!
